base/front50/kustomization.yml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ./base
```

Comment:
I wonder the benefit of this extra layer of indirection for services that don't have multiple variants, but for now it probably makes sense to leave this as is.  Just as we're about to do with `kleat` we can do a code cleanup hackathon on this repo once we have all the services and see where we can clean things up.